test/cli.ts

```diff
@@ -50,6 +52,41 @@ describe('CLI', () => {
       // Defaults to Node.js release type:
       assert.strictEqual(classToRun.releaseType, 'node');
     });
+    it('validates releaseType choices', () => {
```

Comment:
nit, we should probably have a `(done)` callback here, and call it after our assertions, in case the parser were to simply not be called.